src/ResumeAllSession.ts

```diff
@@ -0,0 +1,25 @@
+import { ExtensionContext } from 'vscode';
+import * as vscode from 'vscode';
+
+export class ResumeAllSession {
+    constructor(context: ExtensionContext) {
+        this.registerCommands(context);
+    }
+
+    private readonly registerCommands = (context: ExtensionContext) => {
+        context.subscriptions.push(
+            vscode.commands.registerCommand(
+                'cdt.debug.resumeAllSession',
+                async () => {
+                    const session = vscode.debug.activeDebugSession;
+                    if (session) {
+                        const result = await session.customRequest('cdt-amalgamator/resumeAll');
```

Comment:
When I press resume all I get an error messages (note this appears to be multiple different errors, the logged error in the debug console, plus the pop-up error)

![image](https://user-images.githubusercontent.com/679236/217983047-620e3b28-3b62-4d59-ade0-084ec9ed45d5.png)

<details>

Full log from when I press resume all with both child daps having verbose too:

```
From client: cdt-amalgamator/resumeAll(undefined)
From amalgamator: threads(undefined)
To client: {"seq":113,"type":"event","event":"output","body":{"category":"stdout","output":"From client: threads(undefined)\n"}}
From client: threads(undefined)
To client: {"seq":114,"type":"event","event":"output","body":{"category":"stdout","output":"GDB command: 17 -thread-info\n"}}
GDB command: 17 -thread-info
From amalgamator: threads(undefined)
To client: {"seq":113,"type":"event","event":"output","body":{"category":"stdout","output":"From client: threads(undefined)\n"}}
From client: threads(undefined)
To client: {"seq":114,"type":"event","event":"output","body":{"category":"stdout","output":"GDB command: 17 -thread-info\n"}}
GDB command: 17 -thread-info
To client: {"seq":115,"type":"event","event":"output","body":{"category":"stdout","output":"GDB result: 17 done,threads=[{id=\"1\",target-id=\"process 1820820\",name=\"empty2.exe\",frame={level=\"0\",addr=\"0x0000555555555131\",func=\"function2\",args=[],file=\"empty2.c\",fullname=\"/scratch/debug/git/cdt-amalgamator/sampleWorkspace/empty2.c\",line=\"2\",arch=\"i386:x86-64\"},state=\"stopped\",core=\"0\"}],current-thread-id=\"1\"\n"}}
GDB result: 17 done,threads=[{id="1",target-id="process 1820820",name="empty2.exe",frame={level="0",addr="0x0000555555555131",func="function2",args=[],file="empty2.c",fullname="/scratch/debug/git/cdt-amalgamator/sampleWorkspace/empty2.c",line="2",arch="i386:x86-64"},state="stopped",core="0"}],current-thread-id="1"
To client: {"seq":115,"type":"event","event":"output","body":{"category":"stdout","output":"GDB result: 17 done,threads=[{id=\"1\",target-id=\"process 1820821\",name=\"empty1.exe\",frame={level=\"0\",addr=\"0x0000555555555131\",func=\"function1\",args=[],file=\"empty1.c\",fullname=\"/scratch/debug/git/cdt-amalgamator/sampleWorkspace/empty1.c\",line=\"2\",arch=\"i386:x86-64\"},state=\"stopped\",core=\"9\"}],current-thread-id=\"1\"\n"}}
GDB result: 17 done,threads=[{id="1",target-id="process 1820821",name="empty1.exe",frame={level="0",addr="0x0000555555555131",func="function1",args=[],file="empty1.c",fullname="/scratch/debug/git/cdt-amalgamator/sampleWorkspace/empty1.c",line="2",arch="i386:x86-64"},state="stopped",core="9"}],current-thread-id="1"
To amalgamator: {"seq":0,"type":"response","request_seq":12,"command":"threads","success":true,"body":{"threads":[{"id":1,"name":"empty1.exe","running":false}]}}
To client: {"seq":116,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"response\",\"request_seq\":12,\"command\":\"threads\",\"success\":true,\"body\":{\"threads\":[{\"id\":1,\"name\":\"empty1.exe\",\"running\":false}]}}\n"}}
To client: {"seq":0,"type":"response","request_seq":12,"command":"threads","success":true,"body":{"threads":[{"id":1,"name":"empty1.exe","running":false}]}}
To amalgamator: {"seq":0,"type":"response","request_seq":12,"command":"threads","success":true,"body":{"threads":[{"id":1,"name":"empty2.exe","running":false}]}}
To client: {"seq":116,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"response\",\"request_seq\":12,\"command\":\"threads\",\"success\":true,\"body\":{\"threads\":[{\"id\":1,\"name\":\"empty2.exe\",\"running\":false}]}}\n"}}
To client: {"seq":0,"type":"response","request_seq":12,"command":"threads","success":true,"body":{"threads":[{"id":1,"name":"empty2.exe","running":false}]}}
To client: {"seq":0,"type":"response","request_seq":16,"command":"cdt-amalgamator/resumeAll","success":true,"body":"OK"}
From amalgamator: cdt-gdb-adapter/resumeAll({"threadId":1})
To client: {"seq":118,"type":"event","event":"output","body":{"category":"stdout","output":"From client: cdt-gdb-adapter/resumeAll({\"threadId\":1})\n"}}
From client: cdt-gdb-adapter/resumeAll({"threadId":1})
From amalgamator: cdt-gdb-adapter/resumeAll({"threadId":1})
To client: {"seq":118,"type":"event","event":"output","body":{"category":"stdout","output":"From client: cdt-gdb-adapter/resumeAll({\"threadId\":1})\n"}}
From client: cdt-gdb-adapter/resumeAll({"threadId":1})
To client: {"seq":119,"type":"event","event":"output","body":{"category":"stdout","output":"GDB command: 18 -exec-continue --thread 1\n"}}
GDB command: 18 -exec-continue --thread 1
To amalgamator: {"seq":0,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":120,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"response\",\"request_seq\":13,\"command\":\"cdt-gdb-adapter/resumeAll\",\"success\":true}\n"}}
To client: {"seq":0,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":122,"type":"event","event":"output","body":{"category":"stdout","output":"GDB result: 18 running\n"}}
GDB result: 18 running
To client: {"seq":119,"type":"event","event":"output","body":{"category":"stdout","output":"GDB command: 18 -exec-continue --thread 1\n"}}
GDB command: 18 -exec-continue --thread 1
To amalgamator: {"seq":0,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":120,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"response\",\"request_seq\":13,\"command\":\"cdt-gdb-adapter/resumeAll\",\"success\":true}\n"}}
To client: {"seq":0,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":122,"type":"event","event":"output","body":{"category":"stdout","output":"GDB result: 18 running\n"}}
GDB result: 18 running
To client: {"seq":123,"type":"event","event":"output","body":{"category":"stdout","output":"GDB exec async: running,thread-id=\"all\"\n"}}
GDB exec async: running,thread-id="all"
To client: {"seq":123,"type":"event","event":"output","body":{"category":"stdout","output":"GDB exec async: running,thread-id=\"all\"\n"}}
GDB exec async: running,thread-id="all"
To amalgamator: {"seq":0,"type":"event","event":"continued","body":{"threadId":1}}
To client: {"seq":124,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"event\",\"event\":\"continued\",\"body\":{\"threadId\":1}}\n"}}
To client: {"seq":0,"type":"event","event":"continued","body":{"threadId":1}}
To amalgamator: {"seq":121,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":126,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":121,\"type\":\"response\",\"request_seq\":13,\"command\":\"cdt-gdb-adapter/resumeAll\",\"success\":true}\n"}}
To client: {"seq":121,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":0,"type":"event","event":"continued","body":{"threadId":1001,"allThreadsContinued":false}}
To amalgamator: {"seq":0,"type":"event","event":"continued","body":{"threadId":1}}
To client: {"seq":124,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":0,\"type\":\"event\",\"event\":\"continued\",\"body\":{\"threadId\":1}}\n"}}
To client: {"seq":0,"type":"event","event":"continued","body":{"threadId":1}}
To amalgamator: {"seq":121,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":126,"type":"event","event":"output","body":{"category":"stdout","output":"To client: {\"seq\":121,\"type\":\"response\",\"request_seq\":13,\"command\":\"cdt-gdb-adapter/resumeAll\",\"success\":true}\n"}}
To client: {"seq":121,"type":"response","request_seq":13,"command":"cdt-gdb-adapter/resumeAll","success":true}
To client: {"seq":0,"type":"event","event":"continued","body":{"threadId":1000,"allThreadsContinued":false}}
attempt to send more than one response for command cdt-gdb-adapter/resumeAll

attempt to send more than one response for command cdt-gdb-adapter/resumeAll
```
</details>

Reply:
Sorry, this is my mistake. This error appears when I compare aiming. It should have been:
```
const result = await session.customRequest(
                             'cdt-amalgamator/resumeAll'
                         );
  if (result !== 'OK') {
                             throw new Error('Failed to resume session');
                         }
```
![MicrosoftTeams-image (38)](https://user-images.githubusercontent.com/104537989/218425041-426013ca-8ff1-4092-908d-612213abdeef.png)